utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -199,6 +199,10 @@ def run_indv(self, suffix="base", st_archive=False):
             infostr += ", with restarts every %d %s"%(rest_n, rest_option)
         logger.info(infostr)
 
+
```

Comment:
Yeah, I remember that request. I think @billsacks got most of what he really wanted with the signal handling changes that I put in last week. I'm not adamantly against this TEST_RUN_STATUS concept, but it will certainly require more work and probably deserves its own PR. Adding a new state will impact things like wait_for_tests and test_status.get_overall_test_status and probably others. I'm actually a little shocked and concerned that this change made it through testing without causing any fails.